pkg/cri/server/container_create.go

```diff
@@ -348,3 +349,14 @@ func (c *criService) runtimeSpec(id string, baseSpecFile string, opts ...oci.Spe
 
 	return spec, nil
 }
+
+// Overrides the default snapshotter if Snapshotter is set for this runtime.
+// See See https://github.com/containerd/containerd/issues/6657
+func (c *criService) runtimeSnapshotter(ctx context.Context, ociRuntime criconfig.Runtime) string {
+	if ociRuntime.Snapshotter == "" {
+		return c.config.ContainerdConfig.Snapshotter
+	}
+
+	log.G(ctx).Infof("Set snapshotter for runtime %s to %s", ociRuntime.Type, ociRuntime.Snapshotter)
```

Comment:
make this one debug otherwise it would be spammy.. when set for a runtime..

Reply:
Done